[3.3.5] Commands: lookup player shows which character is online

TheWinchesters opened the pull request:
**Changes proposed:**
-  Show which character is online when using lookup player commands

**Target branch(es):** 3.3.5/master
- [x] 3.3.5
- [ ] master

**Tests performed:** (Does it build, tested in-game, etc.)

Note: The way the code checks if player is online is the best one I found, but not sure if it is the best way.

TheWinchesters committed:
Show 'online' for every online player in lookup player commands

TheWinchesters committed:
Improvements

JoNMii commented:
Don't use comments in Spanish.

TheWinchesters committed:
Indentation

illfated commented:
Cosmetic detail in your PR comment (top):
To make the MarkDown tick box work as intended, remove the blank space after the X,
or save the comment with the box unchecked and put the tick mark in manually.

> **Target branch(es):** 3.3.5/master
> 
> * [X] 3.3.5
> * [ ]  master
>

Shauren reviewed (dismissed):
looks good to me

ccrs commented on sql/updates/world/3.3.5/9999_99_99_world.sql:
```diff
@@ -0,0 +1,2 @@
+-- Show if player is online in lookup player commands
+UPDATE `trinity_string` SET `content_default` = '  %s (GUID %u) |cff00ff00%s|r' WHERE (`entry` = '329');
```
I'm curious, whats the output if the player is offline?

TheWinchesters replied:
I have used the same behavior that the "(Offline)" in the .pinfo command but in this case, it shows the opposite (If player is online).
In the case of .pinf command
```cpp
handler->PSendSysMessage(LANG_PINFO_PLAYER, target ? "" : handler->GetTrinityString(LANG_OFFLINE).....
```
In this case:
```cpp
`handler->PSendSysMessage(LANG_LOOKUP_PLAYER_CHARACTER, name.c_str(), guid, online ? "Online" : "");
```
So, if player is offline, output is "".
Once this shows which player is online, I do not see any reason to show something if player is offline. It would look redundant.

Shauren replied:
Hmm, I take back my approval, make that "Online" string possible to localize (use trinity_string)

ccrs replied:
My questions was actually oriented into what happens to the trinity_string if the given parameter is empty, as in the charater is offline. Server sends the color "pattern" with empty content?

TheWinchesters replied:
I will remove the color when I move it to trinity_string
Is it better?

TheWinchesters commented:
Shauren, I did not like that "online" in that way either, but I am not sure about which `entry` value I should use to move it to trinity_string.

ccrs commented:
either "recycle" one existing or just create a new one

TheWinchesters committed:
Move "oniline" to trinity_string

TheWinchesters commented:
Updated:
- Moved to `trinity_string`
- color was removed
- "Online" now is (online) to match the existing (offline)
- I used the unused entry 48 because it is the most close to (offline) and it looks 'ordered'

jackpoz committed:
Rename 9999_99_99_world.sql to 2019_02_15_00_world.sql

jackpoz commented:
Thanks for the PR :)

TheWinchesters commented:
:)
Thanks to all of you for this project.
